Cleaned up RSS links, updated HTTP to HTTPS where available, and removed links with no HTTPS alternative.

lingcherd opened the pull request:
#### 78d69e2da2f1a5d0d59d1f4dda5e3e27b434dcaa
<pre>
Cleaned up RSS links, updated HTTP to HTTPS where available, and removed links with no HTTPS alternative.
<a href="https://bugs.webkit.org/show_bug.cgi?id=248007">https://bugs.webkit.org/show_bug.cgi?id=248007</a>
&lt;rdar://problem/102446034&gt;

Reviewed by Alexey Proskuryakov.

* Websites/planet.webkit.org/config.ini:

Canonical link: <a href="https://commits.webkit.org/256794@main">https://commits.webkit.org/256794@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/7044b09eef2ffea359f7f43ce674935f9f629117

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/96799 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/6066 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/29892 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/106325 "Built successfully") | 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/6278 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/34795 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/89184 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/103027 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/102474 "Passed tests") | [  ~~🧪 ios-wk2~~](https://ews-build.webkit.org/#/builders/78/builds/4704 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/83412 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/31660 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/86537 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/88406 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/74613 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/99 "Built successfully") | [  ~~🧪 mac-wk1~~](https://ews-build.webkit.org/#/builders/73/builds/19892 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/90 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/21310 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/4881 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/60/builds/43837 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/75/builds/2273 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/1317 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/40603 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

aproskuryakov commented on Websites/planet.webkit.org/config.ini:
```diff
@@ -45,14 +45,14 @@ new_date_format = %B %d, %Y
 # Provide no more than 7 days articles on the front page
 [templates/index.html.tmpl]
-days_per_page = 14
+# Setting this to 10 years
```
This comment says what the following line does, but not why. It's not explained in the commit message either.

aproskuryakov commented on Websites/planet.webkit.org/config.ini:
```diff
@@ -9,17 +9,17 @@
 cache_directory = cache
-new_feed_items = 6
-log_level = DEBUG
+new_feed_items = 500
```
Please add a comment explaining how this number was chosen.

lingcherd replied:
This number has been set on the server since it was brought up in 2017. I suspected it was copied as is from the older server. The new_feed_items setting limits the number of feed that shows up from an site. My best guess is we wanted to display as many feed as possible from any of the subscribed sites, and thus a large number was chosen.

lingcherd committed:
Cleaned up RSS links, updated HTTP to HTTPS where available, and removed links with no HTTPS alternative.
https://bugs.webkit.org/show_bug.cgi?id=248007
<rdar://problem/102446034>

Reviewed by Alexey Proskuryakov.

* Websites/planet.webkit.org/config.ini:

Canonical link: https://commits.webkit.org/256794@main
